algorithms/linfa-logistic/src/lib.rs

```diff
@@ -495,8 +495,7 @@ fn logistic_grad<F: Float, A: Data<Elem = F>>(
     let yz = x.dot(&params.into_shape((params.len(), 1)).unwrap()) + intercept;
     let len = yz.len();
     let mut yz = yz.into_shape(len).unwrap() * y;
-    yz.mapv_inplace(logistic);
-    yz -= F::one();
+    yz.par_mapv_inplace(|v| logistic(v) - F::one());
```

Comment:
Could you put `* y` from below inside par_mapv_inplace? Plus there is `* y` above too.

Reply:
ah, `y` is also a vector, so this would need to be more like map on zipped vectors, store in one of them

Reply:
Maybe another comment, the expression we have right now, isn't it `-logistic(-x)`, so we could avoid possibly numerically bad subtraction.